charts/logging-stack/values.yaml

```diff
@@ -75,6 +43,9 @@ resources:
 input:
   tail:
     memBufLimit: 5MB
+#Set this to cri if you want to collect containerd or cri-o information
```

Comment:
=>
# Set this to cri if you want to collect CRI format logs for containerd or CRI-O.